doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-debugging-tips.rst

```diff
@@ -114,26 +114,32 @@ failure, ask one of the team members for help.
 Debugging an issue using interactive-on-error
 ---------------------------------------------
 
-It is important to be able to reproduce an issue when investigating its cause.
-Run a job similar to the failed job, using the `interactive-on-error`_ mode in
-teuthology::
+When you encounter a job failure during testing, you should attempt to
+reproduce it. This is where ``--interactive-on-error`` comes in. This
+section explains how to use ``interactive-on-error`` and what it does. 
+
+When you have verified that a job has failed, run the same job again in
+teuthology but add the `interactive-on-error`_ flag::
 
     ideepika@teuthology:~/teuthology$ ./virtualenv/bin/teuthology -v --lock --block $<your-config-yaml> --interactive-on-error
 
-For this job, use either `custom config.yaml`_ or the yaml file from
-the failed job. If you intend to use the yaml file from the failed job, copy 
-``orig.config.yaml`` to your local dir and change the `testing priority`_
-accordingly, like so::
+Use either `custom config.yaml`_ or the yaml file from the failed job. If
+you use the yaml file from the failed job, copy ``orig.config.yaml`` to
+your local directory and change the `testing priority`_ accordingly, like
```

Comment:
testing priority doesn't affect manually run jobs like this, it's only used for jobs scheduled by teuthology-suite which adds them to a central queue.

The mention of testing priority should be removed.

Reply:
I've removed the mention of testing priority.